File: src/matlab/save_bin.m
function save_bin(name_,varname,accu)
% ------------------------------------------------------------------------------
% diego domenzain 2021
%
% accu = 'uint32'
% accu = 'double'
%
% 💾:
% save_bin(name_,varname,accu);
%
% 👀:
% var_mat = read_bin(name_,size_,accu);
% ------------------------------------------------------------------------------
fid = fopen(strcat(name_,'.bin'),'w');
fwrite(fid,varname,accu);
fclose(fid);
end

File: src/julia/readwrite/save_bin.jl
function save_bin(name_,varname)
# ------------------------------------------------------------------------------
# diego domenzain 2021
#
# 💾:
# save_bin(name_,varname,accu); 
#
# 👀:
# var_mat = read_binf64(name_,size_);
# ------------------------------------------------------------------------------
fid=open(string(name_,".bin"),"w");
write(fid,varname);
close(fid);
end
